BrainPortal/app/views/tool_configs/_form_fields.html.erb

```diff
@@ -115,5 +115,53 @@
     </p>
   </span>
 
+  <% bourreau = Bourreau.find(@tool_config.bourreau_id) %>
```

Comment:
this will crash completely the view when a ToolConfig doesn't have a bourreau_id, which is legal (tool configs that apply to a tool for all bourreaux)